Gemfile

```diff
@@ -24,6 +24,7 @@ group :development, :test do
   gem 'equivalent-xml'
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
+  gem 'rb-readline' # For readline funkiness on OS X.
```

Comment:
Without it, byebug fails with:
```
     Failure/Error: byebug
     
     LoadError:
       dlopen(/Users/jlittman/.rbenv/versions/2.5.3/lib/ruby/2.5.0/x86_64-darwin18/readline.bundle, 9): Library not loaded: /usr/local/opt/readline/lib/libreadline.7.dylib
         Referenced from: /Users/jlittman/.rbenv/versions/2.5.3/lib/ruby/2.5.0/x86_64-darwin18/readline.bundle
         Reason: image not found - /Users/jlittman/.rbenv/versions/2.5.3/lib/ruby/2.5.0/x86_64-darwin18/readline.bundle
```

Reply:
I've never seen this problem before.  Can you add that to a different PR? At a minimum reference the bug in the comment.

Reply:
I'll remove, but in case you're interested: https://stackoverflow.com/questions/40671803/library-not-loaded-usr-local-opt-readline-lib-libreadline-6-dylib-loaderror